src/util/Constants.js

```diff
@@ -205,6 +205,7 @@ const Endpoints = exports.Endpoints = {
     return {
       Emoji: emojiID => `${root}/emojis/${emojiID}.png`,
       Asset: name => `${root}/assets/${name}`,
+      DefaultAvatar: number => `${root}/embed/avatars/${number}.png`,
```

Comment:
And its correct cause it returns one of the 5 avatars available on cdn.discordapp.com, I don't see why this would be broken 😕